ynput/OpenPype
# Nuke: LoadClip colorspace override - OP-6591

## Changelog Description
Setting the colorspace from the representation data was not supported.

## Testing notes:
1. Setup colormanagement for Nuke.
2. Publish write instance with modified colorspace (`color_picking` for example).
3. Load publish write instance and validate the correct colorspace is set.
4. Add global or host overrided File rules with some other colorspace on your project
5. Load again and notice that the colorspace was overriden (priority on filerules should override colorspaceData)
